src/result.cxx

```diff
@@ -502,6 +502,18 @@ pqxx::row::size_type pqxx::result::columns() const noexcept
 }
 
 
+int const pqxx::result::column_size(pqxx::row::size_type number) const
+{
+  return PQfsize(m_data.get(), number);
```

Comment:
No error handling?  To be honest the libpq documentation is also severely lacking here — it says nothing about how these functions could fail.  If they _really_ couldn't fail, these new methods could be `noexcept`.  But trivially, they can: just feed in an invalid column number.

The fact that nobody's bothered to mention error handling in the libpq documentation, for either of these functions, suggests to me that there is very little demand for them.  Unless you have a concrete need for these, my inclination is to leave them out.

Reply:
Yes, we have a concrete need for them (both PQfsize and PQfmod), that's the reason why we created this PR in the first place. We need to get the the type and the size of a column (especially string columns), just from the result of a query. We do some length checks and we cannot always rely on our internal schema, cause users may create dynamic queries. Rewriting all this to remove the need for dynamic type detection is unfortunately not an option cause this would be way to much work and we absolutely have to support postgres in our next release.
We will try to add everything that's needed to create a PR  that would be acceptable. We just need to figure out the details.

Reply:
Ah, didn't mean to discourage you!  If you have a concrete need, of course that's different.